### ISSUE TITLE
Loss of alignment after saving

### ISSUE THREAD

**Issue by academ88** [ClosedXML/ClosedXML#938]
## Read and complete the full issue template

- [x] Bug

**Version of ClosedXML**

0.93.0

**What is the current behavior?**

I have an excel-template with horizontal alignment = center in cell "Field1" 

![image](https://user-images.githubusercontent.com/40792865/42215102-1d6e64f4-7ec6-11e8-8d53-3294d08fe42f.png)

I've just opened and saved file:
![image](https://user-images.githubusercontent.com/40792865/42215267-90921124-7ec6-11e8-8bc4-945a8a466ef7.png)

Horizontal alignment has been changed.
If I remove the first column, alignment will not be changed.

**What is the expected behavior or new feature?**

Alignment shouldn't be changed

**Did this work in previous versions of our tool?  Which versions?**

Yes/v0.92.1

## Reproducibility

**Code to reproduce problem:**
```c#
XLWorkbook Wb = new XLWorkbook(FileName)
Wb.SaveAs(FileName);
```
[result.xlsx](https://github.com/ClosedXML/ClosedXML/files/2158974/result.xlsx)
[originalFile.xlsx](https://github.com/ClosedXML/ClosedXML/files/2158975/originalFile.xlsx)

**Comment by igitur** [comment]
I'm too lazy to double-check, but are you *sure* this worked in 0.92.1? I think I found the cause of the bug and it's one of those cases where it makes me thing "How did this ever work?"

**Comment by academ88** [comment]
Yes, I'm sure - it worked in 0.92.1.

**Comment by igitur** [comment]
Ah OK, found it. Bug was introduced in 742888cb39849de641f3cd15dc79e8ef8b750e2c.

**Comment by Pankraty** [comment]
The reason is that styles are not propagated to child entities on loading. Therefore, if the alignment is specified on a row level it is not applied to cells. After a workbook is saved the original alignment is lost.

Will fix it a bit later.

**Comment by igitur** [comment]
This is a regression as far as I remember, but I don't recall that we propagated styles to child entities. Maybe I'm wrong...

**Comment by Pankraty** [comment]
No, I was wrong - I thought I had isolated the issue in the test but that was just an incorrect test :)

**Comment by igitur** [comment]
OK. But on that note... I don't think we should ever do such propagation. I believe the current design was intended so that users could finely control where styles are applied.

**Comment by igitur** [comment]
Fixed in #952
